### DISCUSSION TITLE
India Financial Markets Data

### DISCUSSION THREAD

**Discussion by sumanpapanaboina1983** [OpenBB-finance/OpenBB#5234] [General]
Hello,
 First of all thank you for creating such a wonderful opensource project and community.  Would like to know is there any plans on make India Financial Markets Data available as part of OpenBB terminal. Is there are any challenges you are seeing in bringing this data ? or it is already in your roadmap ?

**Comment by IgorWounds** [comment]
Hello,

Thanks for asking this question!

You can already access India Financial Markets Data inside the OpenBB terminal.

For example, you can run the following command:

```
stocks

load --ticker ACHR
```

We even added a custom flag that indicates that you want data from India which can be added after the ticker as such: `--india`.

Hope this helps.

If it does, please feel free to mark this discussion as resolved and close it. If you have more questions, don't hesitate to ask.

**Reply by Janidha** [reply]
Hi,
Thanks for adding --India, any plan or currently way out to use Comparison Analysis. Currently it is only with SPY and cant use it except US stocks.
